models/custom-application/src/custom-application-installation/index.ts

```diff
@@ -0,0 +1,3 @@
+// Export models
+export * as CustomApplicationInstallation from './custom-application-installation';
+export * as RestrictedCustomApplicationInstallationForOrganization from './restricted-custom-application-installation-for-organization';
```

Comment:
Do we really need a model for this? 🤔

Reply:
Actually, no. Removed here [813da3](https://github.com/commercetools/test-data/pull/542/commits/0813da32a05e570724d7fd76637f00f4599896e2).